src/lib/consts.ts

```diff
@@ -1,3 +1,4 @@
 export * from './public-consts';
-export const TRACE_KEY = 'uber-trace-id';
+export const TRACE_KEY = 'traceparent';
```

Comment:
It would be better to re-export this header name from the opentelemetry library
 https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-core/src/index.ts#L79

Reply:
This will require installing the `opentelemetry/core` dependency. Do we really want to add it just for re-exporting a string?

Reply:
@jigetus nope. I thought that we had it.